technical-reports/Performance-Testing.md

```diff
@@ -0,0 +1,105 @@
+# Performance Testing in TEAMMATES
+
+Authors: [Ronak Lakhotia](https://github.com/RonakLakhotia), [Amrut Prabhu](https://github.com/amrut-prabhu) and [Jacob Li PengCheng](https://github.com/jacoblipech)
+
+* [Introduction](#Introduction)
+* [Problem](#Problem)
+* [Overview of the Proposed Solution](#Overview-of-Solution)
```

Comment:
Not necessarily a problem, but the list entry here does not match the actual header.

Reply:
Updated to `Why do we need Performance Testing?`